tools/cmd/release-notes/release-notes.go

```diff
@@ -39,28 +39,37 @@ func getMilestone(title string, client *github.Client) (*github.Milestone, error
 	return nil, errors.New("milestone, " + title + ", not found.")
 }
 
-func getPullRequestIssuesInMilestone(title string, client *github.Client) ([]*github.Issue, error) {
+func getPullRequestIssuesInMilestone(title string, client *github.Client) (map[int64]*github.Issue, error) {
 	milestone, err := getMilestone(title, client)
 	if err != nil {
 		return nil, err
 	}
-	pullRequests, _, err := client.Issues.ListByRepo(ctx, owner, repo, &github.IssueListByRepoOptions{Milestone: strconv.Itoa(milestone.GetNumber()), State: "closed"})
-	if err != nil {
-		return nil, err
-	}
-	result := []*github.Issue{}
-	for _, pullRequest := range pullRequests {
-		if pullRequest.Milestone.GetID() != milestone.GetID() {
-			continue
+	result := map[int64]*github.Issue{}
+	options := github.IssueListByRepoOptions{Milestone: strconv.Itoa(milestone.GetNumber()), State: "closed"}
+	options.Page = 0
+	options.PerPage = 100
```

Comment:
Is the default 30 results/page and max is 100?

Reply:
correct, though the 30 doesn't seem to be documented.